protocol/client/migrations/0001_add_messages_contacts.up.db.sql

```diff
@@ -21,3 +21,9 @@ type INT NOT NULL,
 state INT,
 public_key BLOB
 ) WITHOUT ROWID;
+
+CREATE TABLE IF NOT EXISTS history_topics (
+synced BIGINT,
+contact_id VARCHAR UNIQUE NOT NULL,
+FOREIGN KEY(contact_id) REFERENCES user_contacts(id) ON DELETE CASCADE
```

Comment:
Nice! We definitely should put a comment somewhere where `History` is defined to mention items are deleted because it's not mentioned anywhere.